File: src/main/java/br/com/java/arthurfnsc/MeuPrograma.java
package br.com.java.arthurfnsc;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public final class MeuPrograma {

    private static final Logger LOGGER = LogManager
            .getLogger(MeuPrograma.class);

    private MeuPrograma() {
    }

    public static void main(final String... args) {

        LOGGER.info("Minha primeira aplicação Java!!");
    }
}

File: src/main/kotlin/br/com/kotlin/arthurfnsc/MeuPrograma.kt
package br.com.kotlin.arthurfnsc

import org.apache.logging.log4j.LogManager

class MeuPrograma

val logger = LogManager.getLogger(MeuPrograma::class.java)

fun main(args: Array<String>) {
    logger.info("Minha primeira aplicação Kotlin!!")
}
